nixos/modules/system/boot/plymouth.nix

```diff
@@ -65,6 +66,15 @@ in
         '';
       };
 
+      extraConfig = mkOption {
+        type = types.lines;
+        default = "";
+        description = ''
+          Literal string to append to <literal>configFile</literal>
+          and the config file generated by the plymouth module.
```

Comment:
> and the config file generated by the plymouth module

-> immediately after other options from this module.